logout/logout.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="../clhome.css">
+    
+    <link rel="stylesheet" href="../index.css">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="../login/login.css">
+    <link rel="stylesheet" href="../signup/signup.css">
+    <link rel="stylesheet" href="logout.css">
+    <link rel="stylesheet" href="../cart/cart.css">
+<style>
+    
+</style>
+
+</head>
+<body>
+<div class="container">
```

Comment:
You can set some z-index value to the nav section
![PR review](https://user-images.githubusercontent.com/89376505/154844169-564f7ab6-5c53-436b-bfc8-51c0f1257dfc.JPG)